src/api_call.py

```diff
@@ -0,0 +1,78 @@
+# _*_ Coding: utf-8 _*_
```

Comment:
Is the content of this file substantially different from the one in the main app?  If not, why is it duplicated here?